index.js

```diff
@@ -1 +1,66 @@
-console.log('Hello World!');
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const chai = require('chai');
+
+const should = chai.should();
+
+function checkDataIntergrity(data) {
+  describe('Data Integrity', function() {
+    const {proof} = data;
+    it('"proof" field MUST exist at top-level of data object.', async () => {
+      should.exist(proof);
+      if(!(typeof proof === 'object' || Array.isArray(proof))) {
+        throw new Error('"proof" MUST be an object or an array of objects.');
```

Comment:
so did you try using chai assertions here?

Reply:
Updated to use chai assertions here [8779e57](https://github.com/digitalbazaar/data-integrity-test-suite-assertion/pull/1/commits/8779e5732b2d599e53a2d5efe2c087e6af7ec576)